website/blog/2023-06-04-databend-weekly-96.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "This Week in Databend #96"
+date: 2023-06-04
+slug: 2023-06-04-databend-weekly
+tags: [databend, weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: andylokandy
+  - name: b41sh
+  - name: Chasen-Zhang
+  - name: dantengsky
+  - name: Dousir9
+  - name: everpcpc
+  - name: gitccl
+  - name: JackTan25
+  - name: jun0315
+  - name: leiysky
+  - name: lichuang
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: soyeric128
+  - name: sundy-li
+  - name: tisonkun
+  - name: Xuanwo
+  - name: xudong963
+  - name: zhang2014
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Virtual Column
+
+One way to optimize querying JSON internal fields is to create virtual columns for these fields using `CREATE VIRTUAL COLCUM`. After creating the virtual columns, we can generate them using `GENERATE VIRTUAL COLUMN` function. Once these steps are completed, we can directly access data in the JSON internal fields through their corresponding virtual columns.
```

Comment:
One of the ways to optimize querying JSON internal fields is to create and generate virtual columns for these fields with the `CREATE VIRTUAL COLCUM` and `GENERATE VIRTUAL COLUMN` functions. This enables direct queries against the JSON internal fields through their corresponding virtual columns.